helm/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM intel/dlstreamer:2023.0.0-ubuntu22-gpu682-dpcpp as base
+FROM intel/dlstreamer:2024.1.1-dev-ubuntu22 AS base
```

Comment:
do they have tag related to gpu dpcpp ? or they are all fold into one with dev-ubuntu22 now?

Reply:
I looked and did not find a tag with gpu-dpcpp... I am attempting to test with Arc GPU to verify that it works

Reply:
chat with Brian, and currently this image seems to support our use cases. but it never hurts to test GPU I guess.

Reply:
Yes, yolov5 works with CPU (scope is only yolov5 for now). I added a backlog item for GPU support